tools/stconfig/commands.go

```diff
@@ -51,6 +51,6 @@ func unpackBootBall(bootBall string) (err error) {
 		return err
 	}
 
-	log.Println("Archive unpacked into: " + ball.Dir())
-	return ball.Clean()
+	log.Println("Archive unpacked into: " + ball.Dir() + " Remember to delte folder afterwards")
```

Comment:
delete

but also ... do we really want to leave stuff like this lying around? I have no idea, but telling us to delete it later seems like we don't.

Reply:
Actually I would agree, but why then have an unpack command for stconfig tool, if the function unpacks the bootball in a tmp directory, tells you the tmp dir name  and deleting it right after it.

This seems unlogical.